src/Stryker.Core/Stryker.Core/Initialisation/ProjectFileReader.cs

```diff
@@ -27,22 +27,13 @@ public ProjectFileReader()
             _logger = ApplicationLogging.LoggerFactory.CreateLogger<ProjectFileReader>();
         }
 
-        public ProjectAnalyzerResult AnalyzeProject(string projectFilePath, string solutionFilePath = null)
+        public ProjectAnalyzerResult AnalyzeProject(string projectFilePath, string solutionFilePath)
         {
-            AnalyzerResult analyzerResult = null;
-            AnalyzerManager manager = null;
-
-            if (solutionFilePath is null)
```

Comment:
Using a solution is optional as dotnet core does not need the solution file to build the project. If you have a case where you do need the solution file for a proper dotnet core build, then you will still need to handle cases where a solution file is not given by the user. I would also expect the solution file commandline flag documentation to be updated if a solution file can be used on dotnet core projects.

Reply:
I will handle the case where a solutionfile is not present and change the command line flag documentation to reflect the change.

Reply:
A solutionfile is no longer mandatory, as it was before. Documentation has been updated.